actions/storage.go

```diff
@@ -8,6 +8,7 @@ import (
 	"github.com/gomods/athens/pkg/storage/fs"
 	"github.com/gomods/athens/pkg/storage/mongo"
 	"github.com/spf13/afero"
+	"github.com/gomods/athens/pkg/storage/rdbms"
 )
 
 func newStorage() (storage.Backend, error) {
```

Comment:
after rebase check new signature of this func:
>func newStorage(storageType, storageLocation string) (storage.Backend, error)

so it does not rely on Env variables.